plugin/trino-kafka/src/test/java/io/trino/plugin/kafka/BaseKafkaWithConfluentLatestSmokeTest.java

```diff
@@ -0,0 +1,163 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.kafka;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.io.BaseEncoding;
+import dev.failsafe.Failsafe;
+import dev.failsafe.RetryPolicy;
+import io.trino.Session;
+import io.trino.plugin.kafka.schema.confluent.KafkaWithConfluentSchemaRegistryQueryRunner;
+import io.trino.spi.type.SqlVarbinary;
+import io.trino.testing.BaseConnectorSmokeTest;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.TestingConnectorBehavior;
+import io.trino.testing.kafka.TestingKafka;
+import io.trino.testng.services.ManageTestResources;
+import org.apache.avro.io.BinaryDecoder;
+import org.apache.avro.io.DecoderFactory;
+import org.testng.SkipException;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.time.Duration;
+
+import static io.confluent.kafka.serializers.AbstractKafkaSchemaSerDeConfig.SCHEMA_REGISTRY_URL_CONFIG;
+import static java.lang.String.format;
+import static java.util.Locale.ENGLISH;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.testng.Assert.assertTrue;
+
+public abstract class BaseKafkaWithConfluentLatestSmokeTest
```

Comment:
We use smoke tests if we want to test different deployment (different setup with kafka). Here you are using the same kafka and Trino as in `TestKafkaWithConfluentLatestConnectorSmokeTest`, so I would move all tests to of this class and tests from all implementation of this class to `TestKafkaWithConfluentLatestConnectorTest`